Drop-In/src/main/java/com/braintreepayments/api/FragmentType.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.api;
+
+public enum FragmentType {
```

Comment:
Does this need to be public?

Reply:
Also renamed `FragmentType` to `SelectPaymentMethodChildFragment`.